packages/nameguard-react/src/components/LoadingReportIcon/LoadingReportIcon.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { Tooltip } from "../Tooltip/Tooltip";
+import { ENSName } from "@namehash/ens-utils";
+import { RatingIcon, RatingIconSize } from "../Report/RatingIcon";
+import { RatingLoadingIcon } from "../icons/RatingLoadingIcon";
+
+type LoadingReportIconProps = {
+  onClickOverride?: (ensName: ENSName) => void;
+  size?: RatingIconSize;
+} & React.ComponentProps<typeof RatingIcon>;
+
+export const LoadingReportIcon = ({
+  onClickHandler,
+  size = RatingIconSize.small,
+
+  /*
+    Props are applied to the shield icon which is the onHover trigger element 
```

Comment:
Let's please update this comment to not use words such as "shield icon". Let's please use the exact names of UI components to be more clear and specific.